### PR TITLE
allow to customize result keys

### LINKED ISSUE #15: Allow to customize result keys
At the moment the result object is hardwired with "data" key for result data, and with "pagination" key for pagination data.
It would be useful to be able to customize this keys, in particular:
- allow to change the "data" key (e.g. changing to "results");
- allow to change the "pagination" key, or allow that fields to be directly at the root level of the result object (not nested into a "pagination" key).

I already have a feature branch with this modifications, ready to be pushed to open a PR.... but I'm unable to push (no permissions).
If you're interested let me know.

### PR DESCRIPTION
**Pull request by nicolamoro**
fixes #15

### CODE REVIEW

File: flask_rest_paginate/__init__.py [hunk 1]
```diff
@@ -132,8 +136,12 @@ def _get_page_link(page_number=None):
...
 
-        return {
-            # TODO: use a better name for the pagination object
-            'pagination': OrderedDict(sorted(pagination_schema.items())),
-            'data': schema.dump(items, many=True) if marshmallow else f.marshal(page_obj.items, schema)
-        }
+        ret = {}
+        ret[self._data_object_key] = schema.dump(items, many=True) if marshmallow else f.marshal(page_obj.items, schema)
+        pagination_object = OrderedDict(sorted(pagination_schema.items()))
+        if self._pagination_object_key:
```
Comment (mtShaikh) [inline flask_rest_paginate/__init__.py:142]: I don't understand the purpose of this if condition as the `_pagination_object_key` is being set with a default value `pagination`. The condition here seems unnecessary.

**Review by mtShaikh — CHANGES_REQUESTED** [review]

File: flask_rest_paginate/__init__.py (hunk 1 above)
Comment (nicolamoro) [reply flask_rest_paginate/__init__.py:142]: Hi @mtShaikh and thanks.
The purpose of this condition is to allow that pagination information could be moved directly in the root of the result object (istead of inside pagination key) by setting `_pagination_object_key` to null.
The initial value of `pagination` for that variable is to preserve the actual behaviour by default.

Comment (mtShaikh) [reply flask_rest_paginate/__init__.py:142]: Ahh i see...everything else looks okay
Could also please add an example about this functionality in the example directory and document it in the README as well?
Thanks

Comment (nicolamoro) [reply flask_rest_paginate/__init__.py:142]: Of course.... I'll do as soon as possible!

Comment (nicolamoro) [reply flask_rest_paginate/__init__.py:142]: @mtShaikh readme and examples updated.
Feel free to change any text if not clear enough.
Thanks

**Review by mtShaikh — APPROVED** [review]

**Pull request merged**